specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/secureScore.json

```
@@ -0,0 +1,581 @@
+      "type": "object",
+      "properties": {
+        "max": {
+          "description": "Maximum control score (0..10)",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "maximum": 10,
+          "exclusiveMinimum": false
+        },
+        "current": {
+          "description": "Actual score for the control = (achieved points / total points) * max score. if total points is zeroed, the return number is 0.00",
+          "readOnly": true,
+          "type": "number",
+          "format": "double",
+          "minimum": 0.00,
+          "maximum": 10.00,
+          "exclusiveMinimum": false
+        }
+      }
+    },
+    "SecureScoreControlsList": {
+      "description": "Page of a secure score controls list",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of secure score controls in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoreControlDetails"
+          }
+        },
+        "nextLink": {
+          "description": "The URI to fetch the next page.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SecureScoreControlDetails": {
+      "description": "Secure score control (calculated) object",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecureScoreControlScoreDetails"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecureScoreControlScoreDetails": {
+      "description": "Calculation result data in control level",
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "User friendly display name of the control",
+          "readOnly": true,
+          "type": "string"
+        },
+        "score": {
+          "description": "Actual score object for the control",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecureScoreControlScoreObject"
+        },
+        "healthyResourceCount": {
+          "description": "Number of healthy resources in the control",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "unhealthyResourceCount": {
+          "description": "Number of unhealthy resources in the control",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "notApplicableResourceCount": {
+          "description": "Number of not applicable resources in the control",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "definition": {
+          "$ref": "#/definitions/SecureScoreControlDefinitionItem"
```

Expanding the properties will reduce object complexity
```suggestion
          "$ref": "#/definitions/SecureScoreControlDefinitionItemProperties"
```